### PR TITLE
Organization Settings

### LINKED ISSUE #269: Organization Settings Page Backend
### Overview
Create endpoints for organizations to edit their information.

### Details
- Add primaryColor, secondaryColor, defaultEventState, defaultEventAddress, defaultEventCity, and defaultEventZipCode to the organization data model as strings [these should not be required fields (ie be nullable)].
- Design 1 endpoint that can update these values plus( defaultContactName, defaultContactEmail, defaultContactPhone, name, and website). Slug, invitedAdmins, and active, should not be able to be changed by this endpoint.
- Design 1 endpoint that gets all the data to populate the settings page

### PR DESCRIPTION
**Pull request by ShmulTomer**
Issues I’m having:

- can’t run “npm run build” because prettier is not working, getting “Error: Cannot find module 'prettier-plugin-tailwindcss’” even though I verified that the module is there
- need help applying the theme in _app.jsx (having trouble trying to useSession() to get org data outside of SessionProvider)

Sprint:

- page can be accessed under the settings dropdown on the navbar
- added the new fields
- new sidebar w/ the different sections
- front end using Formik and InputField + corresponding back end
- warning pop-up when leaving the screen
    - for now, shows up even if there are no unsaved changes
- navbar header logo now uses logo link
- sidebar icons as well as save button use the theme colors
- validators for all the fields (not sure what should be required)
- changes to email notifications in mailersend-email.ts (now applies specific org name, event contact email, and BCC’s the notification email) [needs to be tested]
- [TODO, low priority] better way to handle refresh/leaving pop-up when an edit is made
- [TODO, low priority] better icons for sidebar (saw this was under one of the tasks for Sprint Day)

### CODE REVIEW

**Comment by YiQin93** [comment]
This is awesome! Very impressive!! Let's remind ourselves to add error checks/messages as the front-end is implemented.

**Comment by ShmulTomer** [comment]
edited initial message w/ current progress

File: src/utils/mailersend-email.ts [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+import { useEffect, useState } from "react";
+
 const Recipient = require("mailersend").Recipient;
 const EmailParams = require("mailersend").EmailParams;
 const MailerSend = require("mailersend");
 
 export const sendRegistrationConfirmationEmail = async (user, event) => {
+
+  const [orgName, setOrgName] = useState("");
+
```
Comment (colin4554) [inline src/utils/mailersend-email.ts:10]: This is incorrectly calling frontend api instead of just the backend function

**Comment by YiQin93** [comment]
Can't say how much I love this look and feel!! 

I think the default values for event creation are not synced to the actual create an event feature. Having a popup notification as the page is saved would also be wonderful. But other than these, this page is wonderful!!!! Thank you so much for your amazing work!

**Review by colin4554 — CHANGES_REQUESTED** [review]

**Comment by ShmulTomer** [comment]
almost done; light blue theme gives an error

**Pull request merged**